docs/user_guide/build_hdl.rst

```diff
@@ -630,13 +746,49 @@ expect and understand when using ADI HDL repository on these tools.
        (it does on GUI). It will stop at the first version mismatch (a rather
        slow and frustrating process)`
 
+.. list-table:: Tools
+   :widths: auto
+   :header-rows: 1
+
+   * - Notes
+     - Lattice
+   * - Main tools
+     - Radiant
+   * - EDK tools
+     - Propel Builder
+   * - SDK tools
+     - Propel (Eclipse)
+   * - Building library
+     - :red:`Not supported yet.`
+   * - Building the project
+     - Source the system_project_pb.tcl file in Propel Builder tclsh, source the
+       system_project.tcl file in Radiant tclsh after.
+   * - Timing analysis
+     - The projects are usually tested and should be free of timing errors.
+       There is no straightforward method to verify a timing pass (it usually
+       involves writing a TCL proc by itself) on both the tools. The make
+       build will fail and return with an error if the timing is not met.
+   * - SDK (Lattice riscv-rx)
+     - Use the generated sge folder that contains the bsp and the SoC
+       configuration files. You can create a Propel SDK project using the
+       sys_env.xml file. (currently only no-OS and rtos, but not linked yet to
+       ADI no-OS infrastructure)
+   * - SDK (ARM/FPGA combo)
+     - :red:`Not supported or nonexistent yet.``
```

Comment:
Please remove the additional  `

Reply:
Solved in [this](https://github.com/analogdevicesinc/hdl/pull/1270/commits/bad29527ffe5762f129bef38629f114aaf078aab) commit.